src/gui/modals/ControllerConnectionDialog.cpp

```diff
@@ -125,8 +125,7 @@ public slots:
 namespace gui
 {
 
-ControllerConnectionDialog::ControllerConnectionDialog( QWidget * _parent, 
-		const AutomatableModel * _target_model ) :
+ControllerConnectionDialog::ControllerConnectionDialog( QWidget * _parent, const AutomatableModel * _target_model ) :
```

Comment:
```suggestion
ControllerConnectionDialog::ControllerConnectionDialog(QWidget* _parent, const AutomatableModel* _target_model) :
```

Make sure you follow the style guidelines!